github/resource_github_branch_protection_v3_utils.go

```diff
@@ -241,6 +241,7 @@ func flattenAndSetRestrictions(d *schema.ResourceData, protection *github.Protec
 	return d.Set("restrictions", []interface{}{})
 }
 
+// TODO: review pointer structure
```

Comment:
Do you mind explaining what's going on here?

Reply:
This was just a note added while testing and resolved, removing in last commit